### PR TITLE
Update readme files

### PR DESCRIPTION
**Pull request by Tomatetz**

### CODE REVIEW

File: packages/clientele-api/README.md [hunk 1]
```diff
@@ -4,38 +4,222 @@ This API encompasses operations on behalf of your users, which are things like c
...
+  baseURL: "https://api-playground.eu.upvest.co/1.0/",
+  oauth2: {
+    client_id: CLIENT_ID,
+    client_secret: CLIENT_SECRET
   }
-}
+};
+```
+
+## Response objects
```
Comment (alexreichhardt) [inline packages/clientele-api/README.md]: I don't think we have response objects in JS, we're just propagating the response JSON from the Upvest API, right?

File: packages/clientele-api/README.md [hunk 2]
```diff
@@ -4,38 +4,222 @@ This API encompasses operations on behalf of your users, which are things like c
...
+```javascript
+(async () => {
+  let transactions = [];
+  for await (let transaction of this.clientele.transactions.list(WALLET_ID))
+    transactions.push(transaction);
+  console.log("Transactions: ", transactions);
+})();
+```
+
+#### Retrieve specific transaction
```
Comment (alexreichhardt) [inline packages/clientele-api/README.md]: add one more '#' @Tomatetz

File: packages/clientele-api/README.md [hunk 3]
```diff
@@ -4,38 +4,222 @@ This API encompasses operations on behalf of your users, which are things like c
...
 This API client is based on [axios](https://www.npmjs.com/package/axios).
 
-Here is an example how to use it in Node.js:
+Features
+
+- Get user wallets list
+- Create new wallet
+- Retrieve wallet data
+- Get assets list
+- Get asset data
```
Comment (alexreichhardt) [inline packages/clientele-api/README.md]: Change to 'Retrieve asset data'

File: packages/clientele-api/README.md [hunk 4]
```diff
@@ -4,38 +4,222 @@ This API encompasses operations on behalf of your users, which are things like c
...
+- Get transactions list
+- Send transaction
+- Retrieve transaction data by id
```
Comment (alexreichhardt) [inline packages/clientele-api/README.md]: cut the 'by id'

File: packages/tenancy-api/README.md [hunk 5]
```diff
@@ -6,48 +6,142 @@ A tenancy is your "area" of Upvest, where you will be registering your users bef
...
+
+```
+$ npm install @upvest/tenancy-api
+```
+
+In order to retrieve your API credentials for using this client, you'll need to [sign up with Upvest](https://login.upvest.co/sign-up).
+
+### API Keys Authentication
+
+The Upvest API uses the notion of _tenants_, which represent customers that build their platform upon the Upvest API. The end-users of the tenant (i.e. your customers), are referred to as _clients_. A tenant is able to manage their users directly (CRUD operations for the user instance) and is also able to initiate actions on the user's behalf (create wallets, send transactions).
```
Comment (mislav97) [inline packages/tenancy-api/README.md]: users of the _tenants_ are more often referred to as _clientele users_ than _clients_.

File: packages/clientele-api/README.md [hunk 6]
```diff
@@ -4,38 +4,222 @@ This API encompasses operations on behalf of your users, which are things like c
...
+Again, please retrieve your client credentials from the [Upvest account management](https://login.upvest.co/).
 
 ```javascript
-const { UpvestClienteleAPI } = require('@upvest/clientele-api');
+const { UpvestClienteleAPI } = require("@upvest/clientele-api");
 
-const exampleUsername = 'Example User';
-const examplePassword = 'ex@mp1e p@55w0rd';
+const USERNAME = "Example User";
+const PASSWORD = "ex@mp1e p@55w0rd";
```
Comment (mislav97) [inline packages/clientele-api/README.md]: Remove the spaces in the example passwords

**Pull request merged**